docs/storage.md

```diff
@@ -0,0 +1,37 @@
+# :file_cabinet: Storage
```

Comment:
Nice! `storage.get_orders()` can be pretty useful. I have been using it in the past couple of weeks.